### DISCUSSION TITLE
Smaller PixelIT

### DISCUSSION THREAD

**Discussion by NatroNx** [pixelit-project/PixelIt#177] [Q&A] [answered]
Hello, 

I've built a pixelit for my living room and I'm stunned how useful it is and how good it looks.  Now I want more, but I think it is too big for a office desk, where you only sit a meter away. 

Is there a smaller Matrix available and compatible?

**Comment by foorschtbar** [comment]
Hi,

A little bit smaller are the 8x8 Panels you could chain or if u wish a very tiny matrix for your desk, i build just a few days ago a "MicroMatrix" (8x2cm) https://github.com/foorschtbar/MicroMatrix

Regards,
Fabiab

**Reply by o0shojo0o** [reply]
Yes MicroMatrix is the appropriate name 
![image](https://user-images.githubusercontent.com/19333515/176998905-625a99c0-7a37-4e69-a4c4-65f43d145a82.png)

**Comment by NatroNx** [comment, marked as answer]
Hello, 

I've seen this, but not exactly what I'm searchign for. :D It is way too small for my usecase - still cool though.

I'm looking for somthing maybe 2/3 to 1/2 of the size of the original.

**Reply by foorschtbar** [reply]
A little bit smaller (66x66mm per Panel) and "cheap" are the CJMCU 8*8 Panels which are PixelIt Support out of the box: https://s.click.aliexpress.com/e/_DeipLvN

**Reply by NatroNx** [reply]
Will I end up with a small quadratic display, or would I combine for of them and have 32 x 8  - just smaller.  ~ 6,6 x 26,4cm  instead of 8 x 32cm

**Reply by foorschtbar** [reply]
Yes, u should use 4 of them and got a matrix with 264x66mm.

**Reply by NatroNx** [reply]
vielen dank! Ist bestellt.

thx - already ordered

**Reply by foorschtbar** [reply]
Sehr gut, bei fragen meldest du dich hier, im Discord oder bei Telegram.

**Reply by NatroNx** [reply]
vielen Dank ist nun (fast) fertig! Hat wie beschrieben perfekt funktioniert.

https://github.com/o0shojo0o/PixelIt/discussions/48#discussioncomment-3117304

Wie finde ich euch im Discord?

**Reply by o0shojo0o** [reply]
https://discord.gg/JHE9P9zczW

**Reply by miccgn** [reply]
Hi,

I'd love to spend the next winter with building that micro matrix - would be my third PixelIt and looks quite interesting.

You say,
> vielen dank! Ist bestellt.
> 
> thx - already ordered

But where did you order and what? Did you order hundreds of tiny LEDS, and then the PCB with the files provided?
I never created my own PCB so far. Any suggestions as where to order it? Do I really need to solder hundreds of LEDs manually?

Best regards
Michael

**Reply by foorschtbar** [reply]
@miccgn NatroNx ordered the "middle size". The tiny (MicroMatrix) could u find [here (foorschtbar/MicroMatrix)](https://github.com/foorschtbar/MicroMatrix). I soldered the first one by hand (256 LEDs, only possible with a heat plate or reworking station). The other one (which some people here have), i had assembled directly in china. I cant order a single one, but if you are interested, contact me directly via telegram (foorschtbar).

**Comment by miccgn** [comment]
If anybody is interested in ordering the [MicroMatrix](https://github.com/foorschtbar/MicroMatrix)  please let me know (by responding to this message).
It seems the most efficient way is to order pre-assembled PCBs from somebody like JLCPCB in China, which requires a minimum of 5 pieces. Price per piece should be around 40-60 €, approximately.

**Reply by foorschtbar** [reply]
I also added it as Issue to the Repo: https://github.com/foorschtbar/MicroMatrix/issues/1

**Comment by miccgn** [comment]
Meanwhile, @foorschtbar helped me to get a [MicroMatrix](https://github.com/foorschtbar/MicroMatrix).
I added a D1, a BH1750 ambient light sensor, an acrylic front and an acrylic back, and this is the result:

![front](https://user-images.githubusercontent.com/71180164/209484447-b13de28c-29fe-4cd4-b326-7da0edf0767e.jpg)
![back](https://user-images.githubusercontent.com/71180164/209484459-4f87e2ae-411a-4924-b729-71dda16cceed.jpg)
![side](https://user-images.githubusercontent.com/71180164/209484464-6da42376-5a87-4032-9421-1489e582314a.jpg)
![top](https://user-images.githubusercontent.com/71180164/209484477-9a9be4a8-f55e-4b7d-8b03-14079228bae8.jpg)


Thanks @foorschtbar for the great support!
